contactaccessrights/js/src/access-rights/models/location.js

```diff
@@ -0,0 +1,15 @@
+define([
+  'access-rights/modules/models',
+  'common/services/api/api-builder'
+], function (models) {
+  'use strict';
```

Comment:
_(Applies everywhere)_

Leave an empty line between `'use strict'` and the actual code